fictadvisor-web/src/components/common/ui/form/dropdown/Dropdown.tsx

```diff
@@ -16,7 +16,7 @@ import Option from './components/option';
 import * as styles from './Dropdown.styles';
 import { DropDownOption, DropdownProps } from './types';
 
-const Dropdown: FC<DropdownProps> = ({
+const Dropdown: FC<DropdownProps> = memo(function Dropdown({
```

Comment:
why are you memoizing component

Reply:
Because it get rid of unnecessary re-rendenders, and solve the problem of dropdown menu